backend/src/graphql/resolver/TransactionLinkResolver.ts

```diff
@@ -364,6 +393,104 @@ export class TransactionLinkResolver {
     }
   }
 
+  @Authorized([RIGHTS.QUERY_REDEEM_JWT])
+  @Mutation(() => String)
+  async createRedeemJwt(
+    @Arg('gradidoId') gradidoId: string,
+    @Arg('senderCommunityUuid') senderCommunityUuid: string,
+    @Arg('senderCommunityName') senderCommunityName: string,
+    @Arg('recipientCommunityUuid') recipientCommunityUuid: string,
+    @Arg('code') code: string,
+    @Arg('amount') amount: string,
+    @Arg('memo') memo: string,
+    @Arg('firstName', { nullable: true }) firstName?: string,
+    @Arg('alias', { nullable: true }) alias?: string,
+    @Arg('validUntil', { nullable: true }) validUntil?: string,
+  ): Promise<string> {
+    logger.debug('TransactionLinkResolver.queryRedeemJwt... args=', {
+      gradidoId,
+      senderCommunityUuid,
+      senderCommunityName,
+      recipientCommunityUuid,
+      code,
+      amount,
+      memo,
+      firstName,
+      alias,
+      validUntil,
+    })
+
+    const redeemJwtPayloadType = new RedeemJwtPayloadType(
+      senderCommunityUuid,
+      gradidoId,
+      alias ?? firstName ?? '',
+      code,
+      amount,
+      memo,
+      validUntil ?? '',
+    )
+    // TODO:encode/sign the jwt normally with the private key of the sender/home community, but interims with uuid
+    const homeCom = await getHomeCommunity()
+    if (!homeCom.communityUuid) {
+      throw new LogError('Home community UUID is not set')
+    }
+    const redeemJwt = await encode(redeemJwtPayloadType, homeCom.communityUuid)
+    // TODO: encrypt the payload with the public key of the target community
+    return redeemJwt
+  }
+
+  @Authorized([RIGHTS.DISBURSE_TRANSACTION_LINK])
+  @Mutation(() => Boolean)
+  async disburseTransactionLink(
+    @Ctx() _context: Context,
+    @Arg('senderCommunityUuid') senderCommunityUuid: string,
+    @Arg('senderGradidoId') senderGradidoId: string,
+    @Arg('recipientCommunityUuid') recipientCommunityUuid: string,
+    @Arg('recipientCommunityName') recipientCommunityName: string,
+    @Arg('recipientGradidoId') recipientGradidoId: string,
+    @Arg('recipientFirstName') recipientFirstName: string,
+    @Arg('code') code: string,
+    @Arg('amount') amount: string,
+    @Arg('memo') memo: string,
+    @Arg('validUntil', { nullable: true }) validUntil?: string,
+    @Arg('recipientAlias', { nullable: true }) recipientAlias?: string,
```

Comment:
Thanks for the work on this! One thing I'd suggest is to consider consolidating the many function parameters into a single DTO or input object. This will improve readability, reduce the risk of mistakes when passing so many arguments around, and make future maintenance easier.

Having so many individual parameters can get quite verbose and error-prone, especially as the codebase grows. Using an input type would also align better with GraphQL best practices and keep the resolver signature cleaner.